synctl

```diff
@@ -590,10 +593,15 @@ class SyntheticConfiguration(Base):
             "browser": "chrome",
         }
 
-        # self.webpage_action_conf = {}
+        #syntheticType WebpageAction
+        self.webpage_action_conf = {
+            "syntheticType": "WebpageAction",
+            "browser": "chrome",
+            "recordVideo": True
```

Comment:
By default, `recordVideo` should be false. We can remove it as other test definition.